api/turing/service/mlp_service_test.go

```diff
@@ -67,40 +66,37 @@ func TestNewMLPService(t *testing.T) {
 	reset := testSetupEnvForGoogleCredentials(t)
 	defer reset()
 
-	// Create test Google client
-	gc, err := auth.InitGoogleClient(context.Background())
-	require.NoError(t, err)
 	// Create test projects and environments
 	projects := []mlp.Project{{ID: 1}}
 	environments := []merlin.Environment{{Name: "dev"}}
 
 	// Patch new Merlin and MLP Client methods
 	defer monkey.UnpatchAll()
 	monkey.Patch(newMerlinClient,
-		func(googleClient *http.Client, basePath string) *merlinClient {
-			assert.Equal(t, gc, googleClient)
```

Comment:
I removed this comparison because it is now failing due to the differences in the `googleClient.Transport` field, which may now be different, unsurprisingly because these two clients are initialised separately. It isn't immediately clear what we're comparing by having this check too so I just had it removed.